src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -82,7 +82,27 @@ export class PredictiveTextStudioWorkerImpl
     name: string,
     contents: File
   ): Promise<number> {
-    const wordlist = await readExcel(await contents.arrayBuffer());
+    let wordlist: WordList = [];
+    if (/\.(tsv)$/i.test(name)) {
```

Comment:
👍🏼  this would be a good refactor! Not required for this PR though.

Reply:
readTSV does not take an arrayBuffer it takes a string so I had to read the string in addDictionaryToSourceProject. using FileReader. My original PR had concise notation since I just passed a Blob to readTSV but I had to change it to make the ava tests since I could not use Blobs there.